index.js

```diff
@@ -45,20 +66,37 @@ class Aperture {
           return;
         }
 
-        cropArea = `${cropArea.x}:${cropArea.y}:${cropArea.width}:${cropArea.height}`;
+        cropAreaOpts = `${cropArea.x}:${cropArea.y}:${cropArea.width}:${cropArea.height}`;
       }
 
-      const recorderOpts = [
-        this.tmpPath,
-        fps,
-        cropArea,
-        showCursor,
-        highlightClicks,
-        displayId,
-        audioSourceId
-      ];
+      if (IS_MACOS) {
+        const recorderOpts = [
+          this.tmpPath,
+          fps,
+          cropAreaOpts || cropArea,
+          showCursor,
+          highlightClicks,
+          displayId,
+          audioSourceId
+        ];
+
+        this.recorder = execa(path.join(__dirname, 'swift', 'main'), recorderOpts);
+      } else if (IS_LINUX) {
+        const args = ['-f', 'x11grab'];
+
+        if (typeof cropArea === 'object') {
+          args.push(
+            '-video_size', `${cropArea.width}x${cropArea.height}`,
+            '-i', `:0+${cropArea.x},${cropArea.y}`
+          );
+        } else {
+          args.push('-i', ':0');
+        }
+
+        args.push('-framerate', fps, '-draw_mouse', +(showCursor === true), this.tmpPath);
```

Comment:
Did you verify this works? [Based on the documentation](https://ffmpeg.org/ffmpeg-devices.html#Options-19), `draw_mouse` expects a binary value. That's why I'm coercing the boolean to an int.